lms/resources/_js_config/__init__.py

```diff
@@ -78,6 +78,14 @@ def add_document_url(self, document_url) -> None:  # pylint: disable=too-complex
                 "path": self._request.route_path("canvas_api.pages.via_url"),
             }
 
+        elif document_url.startswith("canvas-studio://media"):
```

Comment:
"Media" is the term that Canvas Studio uses to refer to the type of resource in eg. API documentation (https://tw.instructuremedia.com/api/public/docs/#/). I haven't tried this yet, but it looks like it might be possible to upload audio only media.